### PR TITLE
Add pobem to projects.ru.md

### PR DESCRIPTION
**Pull request by belozyorcev**

### CODE REVIEW

**Comment by tadatuta** [comment]
В projects.en.md продублируешь?

**Comment by belozyorcev** [comment]
@tadatuta ок

File: projects/projects.en.md [hunk 1]
```diff
@@ -4,6 +4,33 @@ We have gathered here useful and interesting projects built with BEM.
 
 If your projects are not yet included here, email us at [info@bem.info](mailto:info@bem.info).
 
+## pobem
+
+PostCSS plugin for easy write CSS in BEM notation.
```
Comment (tadatuta) [inline projects/projects.en.md]: I'd suggest something like `PostCSS plugin to abstract from BEM naming scheme: use any delimiters you want`.

Comment (belozyorcev) [inline projects/projects.en.md]: @tadatuta Good ^_^

**Comment by tadatuta** [comment]
@belozyorcev thank you!

**Pull request merged**